indico/MaKaC/webinterface/pages/conferences.py

```diff
@@ -2697,8 +2480,7 @@ def getVars( self ):
 
 class WPConfModifProgram( WPConferenceModifBase ):
 
-    def _setActiveSideMenuItem( self ):
-        self._programMenuItem.setActive()
+    sidemenu_option = 'programme'
```

Comment:
program? (it's using the american spelling in both the WP/RH names and also in the conference sidemenu item name)